src/lib/ajvInstance.ts

```diff
@@ -37,14 +42,34 @@ export const validationErrorToString = async (errors: AjvErrors) => {
   return (
     "Errors:\n\n" +
     errors
-      .filter(({ instancePath }) => {
-        return instancePath !== "";
-      })
-      .map(({ instancePath, params, message }) => {
+      .map(({ instancePath, params, message }, i) => {
         const paramsMessage = yamlDiffPatch("", {}, params);
-        return `${color.yellow(instancePath)} ${
+        const prev = errors[i - 1];
+
+        const isPreviousVersionError =
+          latestConfigVersion !== undefined &&
+          instancePath === "/version" &&
+          message === "must be equal to constant" &&
+          !paramsMessage.includes(latestConfigVersion);
+
+        const isDuplicateError =
+          prev?.instancePath === instancePath &&
+          jsonStringify(prev.params) === jsonStringify(params) &&
+          prev.message === message;
+
+        if (isPreviousVersionError || isDuplicateError) {
+          return "";
+        }
+
+        return `${instancePath === "" ? "" : `${color.yellow(instancePath)} `}${
```

Comment:
Probably `intancePath || color.yellow...`would be simpler? Or you have other reason to use ternary here?

Reply:
Linter forbids using strings in the places where bool is expected to prevent unexpected behavior and to be more explicit about what exactly the check is checking

Reply:
Because e.g. sometimes empty string is valid and you want to get rid of only undefined case